webapp/channels/src/utils/constants/preferences.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export const CATEGORY_CHANNEL_OPEN_TIME = 'channel_open_time';
```

Comment:
Are you changing this so that individual preferences can be imported like `import {CATEGORY_CHANNEL_OPEN_TIME} from 'utils/constants/preferences'`? I'm trying to think if there's a way to do this while just letting us copy and paste from `constants.tsx`